CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+
+Contributing code
+=================
+
+How to contribute
+-----------------
+
+The preferred way to contribute to Kymatio is to fork the
+[main repository](http://github.com/kymatio/kymatio/) on
+GitHub:
+
+1. Fork the [project repository](http://github.com/kymatio/kymatio):
+   click on the 'Fork' button near the top of the page. This creates
+   a copy of the code under your account on the GitHub server.
+
+2. Clone this copy to your local disk:
+
+          $ git clone git@github.com:YourLogin/kymatio.git
+          $ cd kymatio
+
+3. Remove any previously installed versions of Kymatio:
+
+          $ pip uninstall kymatio
+
+and install your local copy with testing dependencies:
+
+          $ pip install -e .[tests]
```

Comment:
shouldn't we recommend `python setup.py develop` here?

Or does this do this?

Reply:
> Note that it is highly recommended to use pip install . (install) and pip install -e . (developer install) to install packages, as invoking setup.py directly will do the wrong things for many dependencies like pulling prereleases and incompatible packages versions and make the package hard to uninstall with pip.

https://stackoverflow.com/a/19048754